houses/index.md

```diff
@@ -1,9 +1,22 @@
 ---
 layout: page
-title: Homeroom
+title: Houses
 ---
 
-## Overview
+Every student is a member of 1 of the 8 Houses. You should be in a public House channel, and you can reference the Notion Dashboard for your House for more informtion.
+
+- [Bhatt House Dashboard](https://turingschool.notion.site/Bhatt-House-Dashboard-ab0bfff9d3c241c4be18cd7196570b7c)
+- [Ellis House Dashboard](https://turingschool.notion.site/Ellis-House-Dashboard-93d6cde95cd24c48a573eb3e15818185)
+- [Golda Ross House Dashboard](https://turingschool.notion.site/Golda-Ross-House-Dashboard-b204adcd212c407aa64d13d0133dc6e1)
+- [Hopper House Dashboard](https://turingschool.notion.site/Hopper-House-Dashboard-eeba713f7199481da62f4a448c941345)
+- [Matsumoto House Dashboard](https://turingschool.notion.site/Matsumoto-House-Dashboard-9c57db95fdae4ec4b38542f5788de384)
+- [Von Anh House Dashboard](https://turingschool.notion.site/Von-Anh-House-Dashboard-34b63458d00847a7a10cfe26f4f33f44)
+- [Windsor House Dashboard](https://www.notion.so/turingschool/Windsor-House-Dashboard-88a5a784888a49888e29d424eb5ed8a0)
+- [Yazzie House Dashboard](https://www.notion.so/turingschool/Yazzie-House-Dashboard-b67b2191f55f4effbd28271f6ba752ee)
+
+
+
+<!-- ## Overview
```

Comment:
I would just axe this stuff, rather than hiding it. It's always there in the git history if we need it, and like this it just clutters things